repositories/mentor_repository.js

```diff
@@ -1,4 +1,5 @@
 const { getKnex } = require("../db");
+const tagsRepo = require("./tags_repository");
```

Comment:
another reason to avoid mixing repositories and leave them with single responsibility is I'm afraid at some point we might end up with circular dependency problem